.circleci/config.yml

```diff
@@ -232,6 +232,29 @@ jobs:
           destination: gs://ecosystem-test-eng-metrics/experimenter/coverage
           extension: json
 
+  check_local_feature_manifests:
+    machine:
+      image: ubuntu-2204:2024.11.1
+      docker_layer_caching: true
+    resource_class: xlarge
```

Comment:
Do we need `xlarge` for this?

Reply:
we do not, that was copypasta from `check_experimenter_x86_64`, removed